app/models/trends/query.rb

```diff
@@ -94,6 +94,13 @@ def perform_queries
     to_arel.to_a
   end
 
+  def language_order_for(trend_class)
+    trend_class
+      .reorder(nil)
```

Comment:
This `reorder(nil)` is needed because all of the initial scope creation starts with `score: :desc` and if we don't remove it, the eventual query does score, then language, then score again - and the results are wrong. Another option here would be to not start the scope with an order, and add the score desc only after this language block is handled.

Reply:
I suppose we could do that in a follow-up PR, yeah.